File: jsnider3/fizzbuzz.hs


fizzbuzz :: Int -> String
fizzbuzz n = case (rem n 3 == 0, rem n 5 == 0) of
  (True, True) -> "FizzBuzz"
  (False, True) -> "Buzz"
  (True, False) -> "Fizz"
  (False, False) -> show n

main :: IO ()
main = do
  mapM_ (putStrLn.fizzbuzz) [1..100]

File: arpitgogia/fizzbuzz.m
function fizzbuzz = main()
    for x = 1:100
      fizzbuzz = '';
      if mod(x, 3) == 0
        fizzbuzz = [fizzbuzz 'Fizz'];
      end
      if mod(x, 5) == 0
        fizzbuzz = [fizzbuzz 'Buzz'];
      end
      if isempty(fizzbuzz)
        disp(x)
      else
        disp(fizzbuzz)
      end
    end
end
main();